app/views/comments/create.js.erb

```diff
@@ -3,8 +3,21 @@
 <% else %>
   $('#comment-<%= @comment.reply_to %>-reply').before('<%= escape_javascript(render :partial => "notes/comment", :locals => {comment: @comment, answer_id: @answer_id}) %>')
 <% end %>
+// create image upload functionality for fresh comment form
+// see editorToolbar.js
+const replyFormBody = $('#comment-form-body-reply-<%= @comment.cid %>');
+const editFormBody = $('#comment-form-body-edit-<%= @comment.cid %>');
+const replyButton = $('#dropzone-small-reply-<%= @comment.cid %>');
+const editButton = $('#dropzone-small-edit-<%= @comment.cid %>');
+const replyFormOptions = getFileUploadOptions(replyFormBody, false);
+const editFormOptions = getFileUploadOptions(editFormBody, false);
+const replyButtonOptions = getFileUploadOptions(replyButton, true);
+const editButtonOptions = getFileUploadOptions(editButton, true);
+replyFormBody.fileupload(replyFormOptions);
+editFormBody.fileupload(editFormOptions);
+replyButton.fileupload(replyButtonOptions);
+editButton.fileupload(editButtonOptions);
```

Comment:
I think `create.js.erb` runs when `comments/create` is called from the controller.

I had a choice between putting this code here, or in `/app/assets/javascripts/comment.js`. The second choice (where `ajax:success` is triggered on adding a comment) seemed more intuitive, but also more complicated to rewrite. Putting this here for now.

Reply:
That note on when/where it is run may be good to include in the file itself in a header comment, how does that sound? Just thinking to help future devs. Thanks!!!